Applications/WebPortal/WebApp/library/source/components/country_selection_box.tsx

```diff
@@ -106,8 +106,8 @@ export class CountrySelectionBox extends React.Component<Properties> {
       backgroundRepeat: 'no-repeat',
       backgroundPosition: 'right 10px top 50%',
       backgroundSize: '8px 6px',
-      '-moz-appearance': 'none' as 'none',
```

Comment:
This stylesheet uses a dynamic style for ```:focus``` on line 112. Will this work without using Aphrodite?